src/components/header-top/header-top.tsx

```diff
@@ -1,11 +1,22 @@
+import { EditMovie } from 'components/edit-movie/edit-movie';
```

Comment:
Well, if we speak about header it means it is always located on top, so, guess `-top` is not required